### ISSUE TITLE
About validate performance

### ISSUE THREAD

**Issue by Mike-Xiao** [mjkwon2021/CAT-Net#48]
Hi,  I'm kind of confused about the validation performance after each training epoch, I am now traing on my own dataset , it seems that the training loss is continuing dropping after each epoch , but the numbers in validation keeps exactly the same after each epoch, but i think it should at least change a bit. may i know what cause that?Thank you!
![image](https://github.com/mjkwon2021/CAT-Net/assets/46700210/da98e8c2-47b1-4315-9a80-9f6174974e54)
![image](https://github.com/mjkwon2021/CAT-Net/assets/46700210/1fde2ea0-ad9c-43b0-83a3-715482528a96)
![image](https://github.com/mjkwon2021/CAT-Net/assets/46700210/c13d4265-5f7b-4e5a-bf52-6d00ffd640f5)
![image](https://github.com/mjkwon2021/CAT-Net/assets/46700210/2850f229-2de5-4c59-bdb2-4c4079b44fa5)

**Comment by Mike-Xiao** [comment]
I just directly used CAT_full.yaml for training settings as suggested in train.py, and at first I thought it was becasues loading the two pretrained models maks the output of validation stable, however after commenting out loading them I still get exactly the same validation numbers after each epoch,  the numbers associated with iou are all close to 50 even at the first epoch,  it bothers me a lot, could you please give me any suggestion about that? thanks alot!
